packages/nodes-base/nodes/Todoist/v2/OperationHandler.ts

```diff
@@ -295,10 +508,13 @@ export class MoveHandler implements OperationHandler {
 
 export class SyncHandler implements OperationHandler {
 	async handleOperation(ctx: Context, itemIndex: number): Promise<TodoistResponse> {
-		const commandsJson = ctx.getNodeParameter('commands', itemIndex) as string;
+		const commandsJson = ctx.getNodeParameter('commands', itemIndex);
+		assertIsString('commands', commandsJson);
+
 		const projectId = ctx.getNodeParameter('project', itemIndex, undefined, {
```

Comment:
this one is weird. Does not seem to be used. Maybe we should remove this? Thoughts @Joffcom ?

Reply:
@mutdmour 🤔 Yeah if we are not using it we might as well remove it, I wonder where it came from

Reply:
Cleaned up https://github.com/n8n-io/n8n/pull/17925/commits/203ac64c5f75fe0b9498c74040c434064dc0108e